changelogs/fragments/gem_module_add_bindir_option.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - gem module - add ``bindir`` option to specify an installation path for executables such as ``/home/user/bin`` or ``/home/user/.local/bin`` (https://github.com/ansible-collections/community.general/pull/2837).
```

```suggestion
  - gem - add ``bindir`` option to specify an installation path for executables such as ``/home/user/bin`` or ``/home/user/.local/bin`` (https://github.com/ansible-collections/community.general/pull/2837).
```
We require a specifier such as that for everything that is *not* a module ;-)